simphony/io/file_lattice.py

```diff
@@ -0,0 +1,157 @@
+from simphony.cuds.lattice import ABCLattice
+from simphony.cuds.lattice import LatticeNode
+from simphony.io.indexed_data_container_table import IndexedDataContainerTable
+from simphony.io.data_container_description import NoUIDRecord
+from simphony.core.data_container import DataContainer
+
+import numpy as np
+
+
+class FileLattice(ABCLattice):
+    """
+    FileLattice object to use in file lattices.
+
+    Attributes
+    ----------
+    file : PyTables file
+        reference to PyTables file where the lattice is or will be located
+    name : str
+        name of the lattice
+    type : string
+        Bravais lattice type (should agree with the _base_vect below).
+    base_vect : D x float
+        defines a Bravais lattice (an alternative for primitive vectors).
+    size : D x size
+        number of lattice nodes (in the direction of each axis).
+    origin : D x float
+        origin of lattice
+    record : tables.IsDescription
+        A class that describes column types for PyTables table.
+
+    Returns:
+    ----------
+    FileLattice
+        The lattice newly added to the file or existing in the file.
+
+    """
+    def __init__(self, file, name, type, base_vect, size, origin,
```

Comment:
Thank you, that's a good point. I'll change this and the related function calls in tests and h5_cuds.get_lattice.